num_of_sectors: 16
default_lowest_point: -0.1